spot_rl_experiments/spot_rl/envs/lang_env.py

```diff
@@ -433,3 +434,4 @@ def step(self, *args, **kwargs):
                 main(spot, args.use_mixer, config, args.output)
             finally:
                 spot.power_off()
+                # TODO: This above line should be spot.shutdown() ????
```

Comment:
I think so!

Reply:
This will be updated in the PR for skill manager -> main